### DISCUSSION TITLE
Plotting Borehole Data with PyVista

### DISCUSSION THREAD

**Discussion by rvg1986** [cgre-aachen/gemgis#319] [Q&A] [answered]
hi all,
i was following the below mentioned tutorial for Plotting Borehole Data with PyVista. 
https://gemgis.readthedocs.io/en/latest/getting_started/tutorial/35_plotting_borehole_data_with_pyvista.html

i am able to execute the code successfully but my requirement is that I need to convert this  3d plotted bore hole tube to be exported in any other format that is compactible with GIS software's or may be in vkt format for paraview visualization. how to accomplish this ?

**Comment by AlexanderJuestel** [comment, marked as answer]
@rvg1986, are you getting a visualization like this: 
![image](https://github.com/cgre-aachen/gemgis/assets/45469915/642c0201-7da3-4c08-9faf-14f820a024e3)

If so, you could just save the object using `tubes[i].save('tube_i.vtk')` where `i` is the index of your single tube in your list of tubes.

**Reply by rvg1986** [reply]
Thanks alexander. That worked fine.
